client/state/data-layer/wpcom-http/index.js

```diff
@@ -21,19 +26,23 @@ import { extendAction } from 'state/utils';
 const fetcherMap = method => get( {
 	GET: wpcom.req.get.bind( wpcom.req ),
 	POST: wpcom.req.post.bind( wpcom.req ),
-}, method, noop );
+}, method, null );
 
 export const successMeta = data => ( { meta: { dataLayer: { data } } } );
 export const failureMeta = error => ( { meta: { dataLayer: { error } } } );
 export const progressMeta = ( { size: total, loaded } ) => ( { meta: { dataLayer: { progress: { total, loaded } } } } );
 
-const queueRequest = ( { dispatch }, action, next ) => {
+export const queueRequest = ( processIngress, processEgress ) => ( { dispatch }, rawAction, next ) => {
```

Comment:
+1 to inbound / outbound. while it is an industry term, we're also not generally network engineers. but, yeah, not really a blocker.

other options: `preprocess` / `postprocess`, `beforeAdvice` / `afterAdvice`...